### PR TITLE
Added title attrs and i18n on meta bar buttons.

### LINKED ISSUE #453: button help on mouseover
there are many buttons, they all need help test on mouseover, i have no idea what they are

screen shot 2015-07-08 at 21 17 43

### PR DESCRIPTION
**Pull request by allcaps**
Fix #453

### CODE REVIEW

**Comment by ghost** [comment]
Can one of the admins verify this patch?

**Comment by gyst** [comment]
Add to whitelist.
Jenkins please test this.

**Comment by gyst** [comment]
test this please test this

File: src/ploneintranet/todo/browser/templates/todo_view.pt [hunk 1]
```diff
@@ -31,19 +31,21 @@
                   </h1>
                   <input name="title" tal:attributes="disabled read_only" type="text" placeholder="Document title" class="doc-title pat-content-mirror" data-pat-content-mirror="target: #document-title" value="${context/title}" />
                   <div class="quick-functions">
-                    <a href="delete_confirmation#content" tal:attributes="href string:${context/absolute_url}/delete_confirmation#content" class="pat-modal icon-trash iconified" i18n:translate="">
+                    <a href="delete_confirmation#content" tal:attributes="href string:${context/absolute_url}/delete_confirmation#content" class="pat-modal icon-trash iconified"  title="Delete this document" i18n:attributes="title" i18n:translate="">
                       Delete
                     </a>
-                    <a href="" class="icon-copy iconified" i18n:translate="">
+                    <a href="" class="icon-copy iconified" title="Copy this document" title="Copy this document" i18n:attributes="title" i18n:translate="">
```
Comment (gyst) [inline src/ploneintranet/todo/browser/templates/todo_view.pt]: You're providing title twice here.

File: src/ploneintranet/workspace/basecontent/templates/document_view.pt [hunk 2]
```diff
@@ -50,7 +50,6 @@
                     <button type="submit" tal:condition="not:read_only" class="icon-floppy">Save</button>
                   </div><!-- quick-functions -->
                 </div> <!-- #meta-bar -->
-                <div tal:condition="python:context.portal_type == 'Document' and not read_only" id="editor-toolbar" class="bar"><p class="loader">Loading…</p></div>
```
Comment (gyst) [inline src/ploneintranet/workspace/basecontent/templates/document_view.pt:54]: Hey! This is what breaks the test!

**Comment by gyst** [comment]
Jenkins please test this

**Comment by gyst** [comment]
Jenkins wake up! Test this please.

**Comment by allcaps** [comment]
Jenkins test this please test.

**Comment by gyst** [comment]
Please test this

**Comment by gyst** [comment]
Jenkins test this please

**Pull request merged**